pennylane/qnodes/passthru.py

```diff
@@ -75,10 +75,11 @@ class PassthruQNode(BaseQNode):
             internally, otherwise convert it into array[float]. Default: True.
     """
 
-    def __init__(self, func, device, **kwargs):
+    def __init__(self, func, device, mutable, **kwargs):
```

Comment:
How come this change is required? In the decorator, you are instantiating the class with `mutable=mutable`, so wouldn't this count as part of `**kwargs`?

Reply:
Yes! This is a subtlety that I have yet to figure out. The reason is, that a `PassthruQNode` is always instantiated with `mutable=True`. However, all the other `QNode` types have this parameter in the signature of their corresponding `__init__` methods. Therefore having a `mutable` parameter would be required for the `PassthruQNode` as well. The disadvantage, in this case, is, however, that this might be misleading.